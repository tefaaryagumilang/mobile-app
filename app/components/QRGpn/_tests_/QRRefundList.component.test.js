import React from 'react';
import renderer from 'react-test-renderer';

import QRRefundList from '../QRRefundList.component';

describe('QRRefundList component', () => {
  it('QRRefundList: renders correctly', () => {
    const tree = renderer.create(<QRRefundList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
