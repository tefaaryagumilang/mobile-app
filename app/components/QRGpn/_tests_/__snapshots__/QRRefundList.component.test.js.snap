// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRRefundList component QRRefundList: renders correctly 1`] = `
<KeyboardAwareScrollView
  extraHeight={120}
  keyboardShouldPersistTaps="handled"
  style={Object {}}
/>
`;
